lib/fs/casefs.go

```diff
@@ -227,6 +158,7 @@ func (f *caseFilesystem) RemoveAll(name string) error {
 }
 
 func (f *caseFilesystem) Rename(oldpath, newpath string) error {
+	f.dropCache(oldpath, newpath)
```

Comment:
What's the reason for this change, i.e. why must the cache not apply to renames?

Reply:
So this is not related to the real issue I was solving, this is me futzing about. I changed the cache time from one second to 15 seconds to make the cache more effective, but I was thinking something that case-only renames would be especially sensitive to this, but really, if that's the case we'll just retry so I think there's no real reason now that I look at it in broad daylight